Tests/AdaptiveStyleTests.swift

```diff
@@ -147,6 +148,58 @@ class AdaptiveStyleTests: XCTestCase {
         }
     }
 
+    @available(iOS 11, *)
+    func testFontMetricsAdaptation() {
+        // Test is still always run (and fails) on the iOS 10 simulator despite
+        // the availability annotation.
+        guard ProcessInfo().operatingSystemVersion.majorVersion > 10 else {
```

Comment:
Open to suggestions here as well. I was unable to figure out how to get Xcode not to run these tests on simulator targets running iOS 10. The `@available` attribute is required or the tests won't even build for iOS 10 targets, but even with the attribute the test is still always executed.

I first tried using an `if #available(...)` instead of checking `ProcessInfo` but that caused a compiler warning for a redundant availability check 🤦‍♂

Reply:
Guessing it’s an Obj-C runtime issue, because dynamic runtime methods are used to figure out which tests to run, and I bet they ignore availability annotations. Probably worth a radar.